model-optimizer/extensions/front/tf/pad_tf_to_pad.py

```diff
@@ -34,6 +34,12 @@ def find_and_replace_pattern(self, graph: Graph):
                 # the input with fill value is an optional third input in TF
                 if not tfpad.in_port(2).disconnected():
                     tfpad.in_port(2).get_connection().set_destination(new_pad.in_port(3))
+                else:
+                    # create Constant node of proper data type (equal to the data type of the Pad first input)
+                    convert_pad_value = create_op_with_const_inputs(graph, ConvertLike, {0: 0.0},
+                                                                    {'name': original_name + '/pad_value_convert'})
+                    convert_pad_value.in_port(1).connect(new_pad.in_port(0).get_source())
+                    new_pad.in_port(3).connect(convert_pad_value.out_port(0))
```

Comment:
@rkazants 
This code was removed in this PR: https://github.com/openvinotoolkit/openvino/pull/6506
For fixing this issue: https://jira.devtools.intel.com/browse/CVS-58776
Is there reason why it was removed?

Reply:
Returning this code back may return the initial issue.

Reply:
@DariaMityagina, according to the specification for Pad: "All new elements are populated with 0 if input not provided." So according to the specification this addition is not needed. 
If you have a plugin issue, please create plugin nGraph transformation to explicitly set fill_value.

This is not a defect of MO